config/plot-config-sample.js

```diff
@@ -567,6 +577,7 @@ var plotConfigs = {
         },
         traces: [
           {
+            hovertemplate_display_pattern: "Creation Date: {{{$row.RCT}}}<br>Vertical",
```

Comment:
For consistency, this example should use 'Horizontal' since that's the orientation of the chart